src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/CommentWrapper.java

```diff
@@ -54,12 +55,45 @@ public static CommentWrapper createNewFromJson(String advisoryId, String newComm
         if (!commentRootNode.has(CommentField.CSAF_NODE_ID.getDbName())) {
             throw new IllegalArgumentException("csafNodeId must be provided!");
         }
+        if (commentRootNode.has(CommentField.ANSWER_TO.getDbName())) {
+            throw new IllegalArgumentException("answerTo must not be provided for comments!");
+        }
         commentRootNode.put(CommentField.ADVISORY_ID.getDbName(), advisoryId);
         commentRootNode.put(CouchDbField.TYPE_FIELD.getDbName(), ObjectType.Comment.name());
 
         return new CommentWrapper(commentRootNode);
```

Comment:
don't use the commentRootNode direct from the input. It has to be validated, that only allowed json values written to the database